test/defn/k-files/rangemap-simple.k

```diff
@@ -0,0 +1,33 @@
+requires "domains.md"
+requires "json.md"
+
+module RANGEMAP-SIMPLE-SYNTAX
+  imports COLLECTIONS
+  imports RANGEMAP
+  imports INT
+  imports JSON
+
+  syntax RangeMap ::= rangemap (Int) [function]
+// ---------------------------------------
+  rule rangemap(0) => .RangeMap
+
+  rule rangemap(1) => range(0, 20) r|-> 40
```

Comment:
We should consider just using standard range syntax like `[0, 20)` instead of `range(0, 20)`